### ISSUE TITLE
What firefox settings do I need to change/disable to make adn work without the yellow banner warning

### ISSUE THREAD

**Issue by KarkanAlzwayed** [dhowe/AdNauseam#1910]
### Describe the issue

I have installed this extension on Firefox and no matter what settings I change, it still complains about conflict with the browser's blockers and whatnot. Which settings on firefox do I need to disable/change to make adn the default and to stop that little yellow message on the settings page?

### Screenshot in which the issue can be seen
![Screenshot from 2021-07-31 12-00-16](https://user-images.githubusercontent.com/49320606/127745620-ba90ac0b-0bf4-47ee-b4b9-b140f4ae40f0.png)

### Steps for anyone to reproduce the issue
I am not sure, but for me, just opening firefox then going to the extension's settings page

### Your settings

- OS/version: Fedora 34 workstation 
- Browser/version: FireFox 90.0.2
- AdNauseam version: 3.11.4
- Other extensions you have installed: I had ublock origin, privacy badger and fingerprint blocker, but removed all of them before running adn
- AdNauseam settings: default. Just flipped on the three first switches

##### Your filter lists

Non, just installed this extension. Whatever default it comes with.
### Your location/country
US

**Comment by dhowe** [comment]
Good suggestion -- let's add a setting to disable warnings for the next release...

**Comment by KarkanAlzwayed** [comment]
So, you're ssying that that warning is a "false positive"?

**Comment by dhowe** [comment]
No, it is a valid warning, but one that advanced users may want to disable

**Comment by dhowe** [comment]
@mneunomne lets try to get this in the next release

**Comment by mneunomne** [comment]
@KarkanAlzwayed this warning you are menitioning is suppossed to show when you have the "strict blocking" settings on in firefox. 

![strictt](https://user-images.githubusercontent.com/4967860/138618950-3418b863-ca95-4320-8153-dd33ff8464c8.PNG)

Is it still showing when you have some different config on ? 

( and yes this PR is only for an option to hide the notifications )

**Comment by KarkanAlzwayed** [comment]
I had it on custom, which I think had some of the "strict" settings. It did hide the warning with strict enable, BTW. I have a couple of questions: After this fix
1. Does having "strict" on defeat the purpose of this extension, or even hinder it from doing what it supposed to do? Is it still ok to keep "strict" on?
2. Is it ok to run this extension alongside privacy badger and uBlock Origin?
